github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-postsubmits.yaml

```diff
@@ -15,7 +15,6 @@ postsubmits:
       grace_period: 5m
     labels:
       preset-dind-enabled: "true"
-      preset-docker-mirror-proxy: "false"
```

Comment:
Bazel is not using it. I don't think this has any effect at all.

Reply:
Here the task for bazel: https://github.com/bazelbuild/rules_docker/pull/1789

Reply:
Any idea why this is happening? https://storage.googleapis.com/kubevirt-prow/logs/push-release-containerized-data-importer-images/1375129174028587008/build-log.txt When I run the same job locally with Phiano it works, and all the pushes complete.

Reply:
let me just check if this nginx  version mentioned in the html is the one from our proxy.

Reply:
ok, at least the response body is definitley from quay.

Reply:
When I talked to quay guys, they said bad request sometimes happens due to excessively large headers. But I don't see how we are creating huge headers.

Reply:
what I find interesting is that a few days ago you had much more errors, right? What did happen to them?

Reply:
I was eating failures at that time, no I abort immediately when I run into an error.

Reply:
To rule out that it has something to do with the repo as such, we could create a `cdi-controller-test` repo and see if it can push there. Would that help a little?

Reply:
Also I modified the script to mimic KubeVirt more closely. Do the core components one push at a time, and have the test containers in a bundle.

Reply:
and try to push `cdi-controller` and another image which worked so far to this new `cdi-controller-test` repo.